proxy-server/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.8-alpine
+
+RUN apk --no-cache add git
+
+WORKDIR /go/src/proxy-main
+ADD ./go/proxy-main.go ./
+
+RUN go get -v
+RUN go install -v .
```

Comment:
Someday, I'd like to convert this to a gomod and freeze off the dependency versions.